### DISCUSSION TITLE
Has TimerX reached a devel halt?

### DISCUSSION THREAD

**Discussion by lg-nineoneninethreenine** [sumeshir26/TimerX#112] [Q&A]
Have we halted development forever?

**Comment by not-nef** [comment]
Everyone just moved on to their own projects i guess

**Reply by lg-nineoneninethreenine** [reply]
I feel distraught

**Comment by sumeshir26** [comment]
Yeah, TimerX got kinda done, and my studies enourmously increased on entering ninth grade.
But, I am working on a kinda cool project in my free time recently, voice assistant which can do many cool things; will make it public soon.

**Reply by lg-nineoneninethreenine** [reply]
> Yeah, TimerX got kinda done, and my studies enourmously increased on entering ninth grade.

Yeah, same. But I can't feel a bit of it. Maybe because I just lose one mark on my math exams haha



> But, I am working on a kinda cool project in my free time recently, voice assistant which can do many cool things; will make it public soon.

Woah! If Carl Pei was impressed, he would hype it up! Can't wait!

**Reply by sumeshir26** [reply]
@im-coder-lg Hey BTW are u in CBSE? (Lol im also a topper in my class: 78/80 for math and 76/80 for science baby!)

**Reply by lg-nineoneninethreenine** [reply]
Well I don't know how the hell I missed this, but yeah, CBSE, topper of the class, genius, and a kind of Benoit Blanc in the class. They ain't scared, I will get the fear in them as well as expose them scandals

**Reply by lg-nineoneninethreenine** [reply]
But I just can't max out on my skills, IDK why.

**Comment by sumeshir26** [comment]
The UI is inspired by GAUDC, LG knows wht I mean

**Reply by lg-nineoneninethreenine** [reply]
Oh, try Material You too!

**Reply by sumeshir26** [reply]
Well, i'm making it Fluent Design tho...

**Reply by lg-nineoneninethreenine** [reply]
Oh... At least it would be better than Cortana, if you get it.


<details> 
<summary>
u get it
</summary>

Cortana ***SUCKS***!


</details>


After a million edits: there we go

**Reply by sumeshir26** [reply]
![image](https://user-images.githubusercontent.com/68823982/193593720-bf8bc44e-e8e7-4af6-a160-afcebb3bd978.png)

**Reply by sumeshir26** [reply]
How's this?

**Reply by sumeshir26** [reply]
The name's temperory btw for obvious reasons

**Reply by lg-nineoneninethreenine** [reply]
Minimal, it is clearly under work. When finely tuned, you can get a proper app.

**Reply by lg-nineoneninethreenine** [reply]
Can you teach me PowerShell? It is better than Windows CMD, and it could help me on my secret feature. I can't do it tomorrow(I think you might get it) but I will give a hint on the feature. It's better than a bunch of frittata commands and is like an ancient 70s program that exists even today.

**Reply by sumeshir26** [reply]
What do you need help for PowerShell on on? Its actuallay pretty easy

**Reply by lg-nineoneninethreenine** [reply]
The commands. They are really weird than CMD or bash, so it's puzzling.

**Reply by sumeshir26** [reply]
No, you can actuallay use CMD or Bash commmands in PS and they still work

**Reply by HuyHung1408** [reply]
wow, these look so cool tbh
